### ISSUE TITLE
Use Nextcloud-stored phone?

### ISSUE THREAD

**Issue by mario** [nextcloud/twofactor_sms#31]
https://github.com/nextcloud/twofactor_sms/blob/46d047082fd7f09c0307fc13a64756e539b1c419/lib/Provider/SmsProvider.php#L102

Could we instead of doing this use phone that user enters in their Profile? Would make it so much easier.

**Comment by ChristophWurst** [comment]
It totally would, yes.

**Comment by mario** [comment]
Let's do that :D Do we need to force the field in Nextcloud to use proper format first though?

**Comment by ChristophWurst** [comment]
Yes, that's the pain point. You don't reliably know whether a phone number is set and the UI also does not mention the number being used for 2FA.

**Comment by ChristophWurst** [comment]
Oh, I think I misread your comment. But yes, we do have to make sure the format suits the messaging back-end. And we have to communicate that the number is used for 2FA as well.

**Comment by ChristophWurst** [comment]
> Oh, I think I misread your comment. But yes, we do have to make sure the format suits the messaging back-end. And we have to communicate that the number is used for 2FA as well.

-> https://github.com/ChristophWurst/twofactor_sms/pull/45
